x-pack/plugins/ml/public/file_datavisualizer/components/analysis_summary/analysis_summary.js

```diff
@@ -32,22 +34,26 @@ export function AnalysisSummary({ results }) {
 
       <EuiDescriptionList
         type="column"
-        listItems={items}
+        listItems={itemsWithI18n}
         className="analysis-summary-list"
+        titleProps={{ tabIndex: '0' }}
       />
     </React.Fragment>
   );
 }
 
+function pimpOutDisplayItemsWithAriaAndI18n(items) {
```

Comment:
Can this function be renamed please.
Maybe something like `addAriaAndI18nToDisplayItems`